scripts_src/maps/argarden.ssl

```diff
@@ -84,79 +82,67 @@ procedure map_update_p_proc begin
    Lighting;
 end
 
+//Extracted procedure to reduce duplication
+procedure Restock begin
+   variable item;
+
+   set_map_var(MVAR_Last_Respawn_Map,game_time);
+   call Make_Geckos;
+   call Make_Plants;
+   if (not(tile_contains_obj_pid(24298,0,PID_XANDER_ROOT))) then begin
+       item:=create_object_sid(PID_XANDER_ROOT,0,0,SCRIPT_AIXANDER);
+       move_to(item,24298,0);
+       set_map_var(MVAR_Current_Xander_Root,map_var(MVAR_Current_Xander_Root)+1);
+   end
+end
+
+procedure Spawn_Type(variable total, variable type, variable object_pid, variable object_script, variable mapvar) begin
+   variable counter:=0;
+   variable item;
+   variable item_hex:=0;
+
+   while (counter < total) do begin
+      Next_Hex;
+      item:=create_object_sid(object_pid,0,0,object_script);
+      while (item_hex <= 0) do begin
+         item_hex:=tile_num_in_direction(map_var(MVAR_Last_Spot),random(0,5),random(0,3));
+      end
+      set_map_var(mapvar,map_var(mapvar)+1);
+      if (type == OBJ_TYPE_CRITTER) then
+         critter_attempt_placement(item,item_hex,0);
+      else if (type == OBJ_TYPE_ITEM) then
+         move_to(item,item_hex,0);
+      counter+=1;
```

Comment:
This is just my preference, but for new/fixed code I add proper spaces around the operators, and usually skip parentheses when they aren't adding any clarity.
```c++
counter += 1;
item_hex := 0;
if type == OBJ_TYPE_CRITTER then
```
Not insisting, but I think it makes code more readable.

Reply:
Yeah, that's what I wanted to talk about: better set some coding standards for pull requests, cause I prefer adding spaces too.

Reply:
Well, there's no standard policy, because there was no need for it, because this is the very first pull request.
If you want to discuss that, see the forum.
Until there's something set in writing, I'll just use my best judgement of what is too ugly.